sentry_sdk/scope.py

```diff
@@ -423,6 +459,7 @@ def __copy__(self):
         rv._span = self._span
         rv._session = self._session
         rv._force_auto_session_tracking = self._force_auto_session_tracking
+        rv._attachments = list(self._attachments)
```

Comment:
Noting that this is a shallow-copy (and so is contexts and extras) -- changes to an attachment in one scope applies globally.

Reply:
Yeah. I was not sure yet what a good plan is here. I would prefer we force the attachment objects to be immutable (hard to do in Python).

Reply:
Going to leave as such for now.